exonum/src/blockchain/config.rs

```diff
@@ -50,6 +50,22 @@ pub struct ValidatorKeys {
     pub consensus_key: PublicKey,
     /// Service key is used to sign transactions broadcast by the services.
     pub service_key: PublicKey,
+
+    /// No-op field for forward compatibility.
+    #[protobuf_convert(skip)]
+    #[serde(default, skip)]
+    non_exhaustive: (),
+}
+
+impl ValidatorKeys {
+    /// Creates a new `ValidatorKeys` object.
+    pub fn new(consensus_key: PublicKey, service_key: PublicKey) -> Self {
```

Comment:
This method will break if new fields are added to `ValidatorKeys`, right? Should it be documented as unstable then?